src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Infrastructure/OuterApi/IOuterApiService.cs

```diff
@@ -33,6 +33,7 @@ public interface IOuterApiService
         Task<GetPriorLearningSummaryQueryResult> GetPriorLearningSummary(long providerId, long cohortId, long apprenticeshipId);
         Task<GetCohortDetailsResponse> GetCohortDetails(long providerId, long cohortId);
         Task<long> CreateFileUploadLog(long providerId, IFormFile attachment, List<CsvRecord> csvRecords);
+        Task<long> UpdateFileUploadLog(long providerId, long logId);
```

Comment:
I don't think we are using this code yet, and writing code before it's needed causes confusion in this already messy codebase. We may never need this endpoint depending on how we implement that unhandled error exception. Also the parameters here are just identity params and there is no update content. Would it be easier just to remove it

Reply:
hi paul, removed all the update endpoint stuff now